src/test/resources/specs/name-matching/Last_names_combination.feature

```diff
@@ -0,0 +1,182 @@
+Feature: Last name combinations
+
+  @name_matching
+  Scenario: Multi-part last name is tried without splitting
+    Given HMRC has the following individual records
+      | First name | Last name | Date of Birth | nino |
+      | Any        | Any       | 0001-01-01    | Any  |
+    When an income request is made with the following identity
+      | First name    | Arthur       |
+      | Last name     | Brian Coates |
+      | Date of Birth | 9999-12-31   |
+      | nino          | NR123456C    |
+    Then the following names will be tried
+      | First name  | Last name     |
+      | Arthur      | Brian Coates  |
+
+
+  @name_matching
+  Scenario: Multi-part last name is tried with splitting
+    Given HMRC has the following individual records
+      | First name | Last name | Date of Birth | nino |
+      | Any        | Any       | 0001-01-01    | Any  |
+    When an income request is made with the following identity
+      | First name    | Arthur       |
+      | Last name     | Brian Coates |
+      | Date of Birth | 9999-12-31   |
+      | nino          | NR123456C    |
+    Then the following names will be tried
+      | First name  | Last name     |
+      | Arthur      | Coates        |
+      | Brian       | Coates        |
+      | Coates      | Arthur        |
+      | Coates      | Brian         |
+      | Brian       | Arthur        |
+
+
+  @name_matching
+  Scenario: Multi-part first name is tried without splitting
```

Comment:
In the candidate generators multi-part means more than one word separated by spaces. Although the result will be the same, I think this test would be better if "Arthur-Brian" was "Arthur Brian", unless there's a reason for hyphenating?

If that's how it was in the unit test then maybe the title should be changed, and maybe we should test both cases.

Reply:
Modifying the terminology. Split first name and last name tests into separate feature files.

Reply:
I've clarified the test name and what it tests